projects/plugins/jetpack/modules/shortcodes/youtube.php

```diff
@@ -369,6 +371,22 @@ function youtube_id( $url ) {
 
 	}
 
+	/**
+	 * Format output for Calypso Reader/Notifications/Comments
+	 */
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+		require_once WP_CONTENT_DIR . '/lib/display-context.php';
+		$context = A8C\Display_Context\get_current_context();
+		if ( A8C\Display_Context\NOTIFICATIONS === $context ) {
+			return sprintf(
+				'<a href="%1$s" target="_blank" rel="noopener noreferrer"><img src="%2$s" alt="%3$s" /></a>',
+				esc_url( add_query_arg( 'v', $id, 'https://www.youtube.com/watch' ) ),
```

Comment:
Should this be extracted as well, since it's also used above, just where you've extracted the thumbnail? What do you think?

Reply:
Thanks @jeherve . I didn't think to check if the URL was being used in other places as well. Updated this but... I think I made a mess of the branch during my push. I am not exactly sure what happened. Any thoughts on how to go about resolving this?